ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -778,6 +778,10 @@ private void SensorMonitor()
             bool elTempSafe = checkTemp(currElTemp, true);
             bool azTempSafe = checkTemp(currAzTemp, true);
 
+            // Get initial motor and absolute encoder values
+            Orientation currentABSPosition = RadioTelescope.SensorNetworkServer.CurrentAbsoluteOrientation;
+            Orientation currentMotorPosition = RadioTelescope.PLCDriver.GetMotorEncoderPosition();
```

Comment:
Instead of using `GetMotorEncoderPosittion()`, we should be using `GetCurrentOrientation()`. This is because `GetCurrentOrientation()` applies the "Final Position Offset" which is the offset we use to translate the direct motor encoder readings to the telescope's actual orientation.

- [ ] Switch to `GetCurrentOrientation()`